crates/ruff_linter/src/checkers/ast/analyze/expression.rs

```diff
@@ -1301,6 +1301,9 @@ pub(crate) fn expression(expr: &Expr, checker: &mut Checker) {
             if checker.enabled(Rule::IfExprWithTwistedArms) {
                 flake8_simplify::rules::twisted_arms_in_ifexpr(checker, expr, test, body, orelse);
             }
+            if checker.enabled(Rule::IfExprMinMax) {
+                refurb::rules::if_expr_min_max(checker, expr, test, body, orelse);
```

Comment:
Can you instead pass in the `ast::ExprIfExp` here? That's what we prefer for new rules. So e.g. on line 1284, you'd change to:

```rust
Expr::IfExp(if_exp @ ast::ExprIfExp {
    test,
    body,
    orelse,
    range: _,
}) => {
    ...
}
```

And then here, you'd pass in `if_exp` instead of the destructured fields. That way, callers can't pass in the "wrong" values.

Reply:
Good point!